client/components/ProjectSkills.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components/native';
+import { Entypo } from '@expo/vector-icons';
+
+
+const ProjectSkills = () => {
+
+    const SkillsView = styled.View`
+    display: flex;
+    flex-direction: row;
+    margin-top: 17px;
+    
+    `
+
+    const SkillText = styled.Text`
+    border: ${props => `2px solid ${props.theme.colors.staBlack}`};
+    border-radius: 4px;
+    margin-left: 8.5px;
+    padding: 0px 1px 0px 1px;
+    `
```

Comment:
styled components should be outside of the component definition as they need to be static